src/main/java/teammates/ui/template/FeedbackSessionsList.java

```diff
@@ -0,0 +1,18 @@
+package teammates.ui.template;
+
+import java.util.List;
+
+public class FeedbackSessionsList {
+    
+    private List<FeedbackSessionRow> existingFeedbackSessions;
+    
+    public FeedbackSessionsList(List<FeedbackSessionRow> existingFeedbackSessions) {
+        this.existingFeedbackSessions = existingFeedbackSessions;
+    }
```

Comment:
I was initially going to add the attributes of the table here (css classes, column names, onclick handlers ...), but didn't do that in end. Having a class for this could still be useful in future for adding more buttons or behaviour in future